presidio-anonymizer/presidio_anonymizer/entities/analyzer_results.py

```diff
@@ -0,0 +1,63 @@
+"""Wrapper to the list of AnalyzerResult which manipulates the list."""
+from typing import List
+
+
+class AnalyzerResults(List):
+    """
+    Receives the analyzer result list and manipulates it.
+
+     The manipulation contains removal of unused results and sort by indices order.
```

Comment:
```suggestion
     For example:  removal of unused results and sort by indices order.
```

Reply:
it's not an example. that's what it contains.

Reply:
Let me rephrase: The word manipulation is not a good fit here. You can say "It includes ...".
In general it's hard to read this documentation